test/kfp-functional-test/kfp-functional-test.sh

```diff
@@ -12,11 +12,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-sudo apt-get update -y
-sudo apt --no-install-recommends -y -q install curl
-source_root="$(pwd)"
 
+source_root="$(pwd)"
 python3 -m pip install -r "${source_root}/test/kfp-functional-test/requirements.txt"
-HOST="https://$(curl https://raw.githubusercontent.com/kubeflow/testing/master/test-infra/kfp/endpoint)"
 
-python3 "${source_root}/test/kfp-functional-test/run_kfp_functional_test.py" --host "${HOST}"
+if [ -n "$KIND_REGISTRY" ]; then
+    # if KIND_REGISTRY is available, we're running GH actions, host is not required
+    python3 "${source_root}/test/kfp-functional-test/run_kfp_functional_test.py" --host ""
+else
```

Comment:
We expect to remove the Prow jobs to run these tests, I think this confitional is not required.